recipes/tiny-aes-c/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,96 @@
+#include <stdio.h>
```

Comment:
It's exactly same file. You could add C++ method (e.g cout) to link libstdc++

Reply:
Not exactly, it includes _aes.hpp_ instead of _aes.h_